feature/session/src/test/java/io/github/droidkaigi/confsched2020/session/ui/viewmodel/SessionsViewModelTest.kt

```diff
@@ -26,12 +27,14 @@ class SessionsViewModelTest {
     @get:Rule val viewModelTestRule = ViewModelTestRule()
     @get:Rule val mockkRule = MockkRule(this)
     @MockK(relaxed = true) lateinit var sessionRepository: SessionRepository
+    @MockK lateinit var sessionAlarm: SessionAlarm
 
     @Test
     fun load() {
         coEvery { sessionRepository.sessionContents() } returns flowOf(Dummies.sessionContents)
         coEvery { sessionRepository.refresh() }
-        val sessionsViewModel = SessionsViewModel(sessionRepository = sessionRepository)
+        val sessionsViewModel =
+            SessionsViewModel(sessionRepository = sessionRepository, sessionAlarm = sessionAlarm)
```

Comment:
Can you use this form? 🙏 
https://developer.android.com/kotlin/style-guide#functions

```kotlin
val sessionsViewModel = SessionsViewModel(
    sessionRepository = sessionRepository, 
    sessionAlarm = sessionAlarm
)
```

Reply:
Oh, I didn't know style guide. 😅
Of course I do !